core/trino-main/src/main/java/io/trino/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -297,7 +297,9 @@ protected Node visitShowRoles(ShowRoles node, Void context)
 
             if (node.isCurrent()) {
                 accessControl.checkCanShowCurrentRoles(session.toSecurityContext(), catalog);
-                List<Expression> rows = metadata.listEnabledRoles(session, catalog.orElseThrow()).stream()
+                Set<String> enabledRoles = catalog.map(c -> metadata.listEnabledRoles(session, c))
```

Comment:
catalog is present as it is checked above you can simply ` metadata.listEnabledRoles(session, catalog.get))`

Reply:
The check above should have been removed.  I think it was a rebase conflict